### PR TITLE
Cut v2.33.0

### PR DESCRIPTION
**Pull request by beorn7**
The only difference to 2.33.0-rc.1 is a bump in the prometheus/client_golang dependency to 1.12.1. This bump fixes a data race and reduces the excessive amount of histogram buckets in the Go runtime metrics, both introduced by client_golang 1.12.0, which has been used for the first time in 2.33.0-rc.1. I think these changes are low-risk enough to not release them in a rc.2. To reduce friction and user confusion (another RC just for a dependency bump, which we usually don't even acknowledge in the CHANGELOG), I propose to go for the final release now.

However, I can see how you might come to a different conclusion and would like to hear your concerns, if there are any.

### CODE REVIEW

**Review by kakkoyun — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,10 +1,4 @@
-## 2.33.0-rc.1 / 2022-01-19
-
-* [ENHANCEMENT] UI: Debounce search bar. #10157
```
Comment (LeviHarrison) [inline CHANGELOG.md:3]: Why isn't this included in the final changelog? I think it's an independent change.

**Comment by LeviHarrison** [comment]
> I propose to go for the final release now.

Agreed.

File: CHANGELOG.md (hunk 1 above)
Comment (roidelapluie) [reply CHANGELOG.md:3]: it's not, it's part of  #10131 and #10103

**Review by roidelapluie — APPROVED** [review]
LGTM!

**Review by csmarchbanks — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (LeviHarrison) [reply CHANGELOG.md:3]: Oh ok

Comment (beorn7) [reply CHANGELOG.md:3]: Yeah, so essentially, if we further enhance features in a RC that were just added with this release, we do not list the feature and its enhancement separately, but just the feature.

**Comment by beorn7** [comment]
Thanks, everyone.

**Pull request merged**